pkg/test/echo/client/parsedresponse.go

```diff
@@ -173,6 +176,25 @@ func (r ParsedResponses) CheckHostOrFail(t test.Failer, expected string) ParsedR
 	return r
 }
 
+// CheckMTLS asserts that mutual TLS was used.
+// Note: this only is detectable for *successful* HTTP based traffic. Other types will always pass
```

Comment:
Should the function name reflect that this function is only for HTTP based traffic? E.g., CheckMTLSHttPTraffic().

Reply:
done